pkg/agent/agent.go

```diff
@@ -755,12 +761,18 @@ func (i *Initializer) setupDefaultTunnelInterface() error {
 		}
 		tunnelPortUUID, err := i.ovsBridgeClient.CreateTunnelPortExt(tunnelPortName, i.networkConfig.TunnelType, config.DefaultTunOFPort, shouldEnableCsum, localIPStr, "", "", "", nil, externalIDs)
```

Comment:
Although the OFPort is fixed in this PR, I suggest to add log for created OFPort ID, because users need to know the ID if the ID is not a fixed value.